test/functional/http/post/2.delegate.js

```diff
@@ -198,8 +198,15 @@ describe('POST /api/transactions (type 2) register delegate', () => {
 		phases.confirmation(goodTransactions, badTransactions);
 	});
 
+	// TODO: These test cases broke after removing support for batch HTTP POST
+	// transactions and only allowing a single POST transaction per request.
+	// The issue doesn't appear to be related. Maybe a race condition?
+	// See https://github.com/LiskHQ/lisk/issues/1427
```

Comment:
These TODO comments are lo longer needed - The tests are all passing after rebase. I can remove the comments in my next PR.

Reply:
I'll fix in this PR since there is more feedback to resolve.